DBConnection/loadPrice.js

```diff
@@ -28,6 +28,34 @@ const loadMarkPricebySymbol = async (market, pool) => {
   }
 }
 
+const loadIncomingPrice = async (pool) => {
+  try{
+    for(let i = 0; i < symbols.length; i++){
+      await loadIncomingkPricebySymbol(symbols[i], pool);
+    }
+  } catch (err) {
+    return console.log("loadIncomingPrice err", err);
+  }
+} 
+
+const loadIncomingkPricebySymbol = async (market, pool) => {
+  try{
+    const {price, timestamp: time} = await priceGetter(market, dbInput.incomingPriceTimeframe);
```

Comment:
I think promise.all would be faster here since requests will fire concurrently. 

something like 
```
const results = await Promise.all(arrayOfPromise)
results.forEach(result=> writeToDB(result))
```